Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -212,6 +222,22 @@ const ContestsProvider = ({ children }: IContestsProviderProps) => {
         [ currentPage, filters, sortingTypes ],
     );
 
+    const getContestByProblemId = useCallback((problemId: number) => {
+        if (isNil(problemId)) {
```

Comment:
This check is repeated in `SubmissionDetails.tsx`. Remove it from there.

Reply:
I could not find where this check is being done in SubmissionDetails.tsx. I am checking there on row 38 only if the currentSubmission isn't null. If I do not do that, then I get a nullability error when destructuring the problem id.